WcaOnRails/spec/views/competitions/my_competitions.html.erb_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe "competitions/my_competitions" do
+  let(:competition) { FactoryGirl.create(:competition, :registration_open, name: "Melbourne Open 2016") }
+  let(:registration) { FactoryGirl.create(:registration, competition: competition) }
+
+  before do
+    allow(view).to receive(:current_user) { registration.user }
+    assign(:not_past_competitions, [competition])
+    assign(:past_competitions, [])
+  end
+
+  it "shows upcoming competitions" do
+    render
+    expect(rendered).to match /<td><a href="\/competitions\/MelbourneOpen2016">Melbourne Open 2016<\/a><\/td>/
```

Comment:
Rubocop didn't like the escaping and suggested %r which is what I changed it to.